src/coreComponents/physicsSolvers/solidMechanics/SolidMechanicsLagrangianFEM.cpp

```diff
@@ -1049,9 +1026,7 @@ SolidMechanicsLagrangianFEM::
                                          parallelDevicePolicy< 32 > >( targetSet,
                                                                        time_n + dt,
                                                                        targetGroup,
-                                                                       keys::TotalDisplacement, // TODO fix use of
-                                                                       // dummy
-                                                                       // name
+                                                                       extrinsicMeshData::solidMechanics::totalDisplacement::key(),
```

Comment:
It's no more a "dummy name"?

Reply:
I put back the comment before the function call